parlai/tasks/eli5/README.md

```diff
@@ -0,0 +1,143 @@
+# ELI5
+**Adapted from Facebook Research's ELI5 found at https://github.com/facebookresearch/ELI5/**
+
+![GitHub license](https://img.shields.io/badge/license-BSD-blue.svg)
+
+
+Read the Paper: https://arxiv.org/abs/1907.09190
+
+Read the Blog Post: https://ai.facebook.com/blog/longform-qa/
+
+Explore the Dataset: https://facebookresearch.github.io/ELI5/explore.html
+
+--------------------------------------------------------------------------------
+
+# Data creation
+
+We provide a suite of scripts to download paired questions and answers from the ELI5 subreddit along with supporting documents from the CommonCrawl
+
+### FAQ: can you provide the processed data?
+
+No, we are not allowed to host processed Reddit or CommonCrawl data. While we are aware that it would make life a little easier, making your own version of the dataset following the instructions here is pretty straightforward if you have access to a SLURM cluster. **We're also happy to work with you if your cluster works on another operating system, open an issue and we'll get on it!** Since the creation process involves downloading, filtering and tokenizing a full CommonCrawl dump however, using a single machine is impractical.
```

Comment:
we should get rid of this FAQ - "we" is referring to the ELI5 repo owners, but we as ParlAI are not speaking on their behalf. If anything, please direct people to the ELI5 repository for help with cluster stuff etc.